packages/mongodb-rag-core/src/contentStore/MongoDbEmbeddedContentStore.ts

```diff
@@ -95,10 +95,12 @@ export function makeMongoDbEmbeddedContentStore({
       return await embeddedContentCollection.find(pageIdentity(page)).toArray();
     },
 
-    async deleteEmbeddedContent({ page, dataSources }) {
+    async deleteEmbeddedContent({ page, dataSources, inverse = false }) {
```

Comment:
please test this added functionality

Reply:
added the test. Do you think its awkward to have inverse only be applied to dataSources? Should it be an option for pages as well?

Reply:
see below suggestion for this